edge-hub/test/Microsoft.Azure.Devices.Routing.Core.Test/query/ConditionCompilerTest.cs

```diff
@@ -602,7 +602,8 @@ static class TestSuccessDataSource
                 new object[] { "as_number(\"3\") = 3", Bool.True },
                 new object[] { "AS_NUMBER(\"3\") = 3", Bool.True },
                 new object[] { "AS_NUMBER('10e10') = 10e10", Bool.True },
-                new object[] { "AS_NUMBER('10e1000') = 1/0", Bool.Undefined },
+                new object[] { "AS_NUMBER('10e1000') = 10e1000", Bool.True },
+                new object[] { "AS_NUMBER('10e1000') = 1/0", Bool.True },
```

Comment:
10e1000 throws exception in .Net Core 2.2, but it doesn't in .Net Core 3.1.  For details, read https://docs.microsoft.com/en-us/dotnet/core/compatibility/2.2-3.1#floating-point-parsing-operations-no-longer-fail-or-throw-an-overflowexception